modules/optimization/test/tests/executioners/constrained/inequality/main_auto_adjoint.i

```diff
@@ -0,0 +1,77 @@
+# This tests that a linear and constant function can be scaled in
```

Comment:
This comment to should be changed to something like this and also include something about the constraint.

```suggestion
# This tests that a linear function can be scaled in
# functionNeumannBCs applied to the left sideset using
# a parsed functions.  The scale of the linear function
# are being parameterized.
```